hail/python/hail/docs/experimental.rst

```diff
@@ -36,6 +36,7 @@ Genetics Methods
     load_dataset
```

Comment:
Where's export_entries_by_col?

Reply:
oh no! I started doing this first, then realized I should do the rename first, and then forgot the rst.